### PR TITLE
V4 Azure functions + some tweaks 

### PR DESCRIPTION
**Pull request by Abrhm7786**
### PR checklist
 
- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] Pull Request title clearly describes the work in the pull request and Pull Request description provides details about how to validate the work. Missing information here may result in delayed response from the community.
- [x] Run the following to [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) and update samples:
  ```
  ./mvnw clean package 
  ./bin/generate-samples.sh
  ./bin/utils/export_docs_generators.sh
  ``` 
  Commit all changed files. 
  This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. 
  These must match the expectations made by your contribution. 
  You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/configs/java*`. 
  For Windows users, please run the script in [Git BASH](https://gitforwindows.org/).
- [x] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master` (5.3.0), `6.0.x`
- [x] If your PR is targeting a particular programming language, @mention the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) members, so they are more likely to review the pull request.

### CODE REVIEW

File: docs/generators/csharp-netcore-functions.md [hunk 1]
```diff
@@ -14,6 +16,7 @@ title: Documentation for the csharp-netcore-functions Generator
 
 ## CONFIG OPTIONS
 These options may be applied as additional-properties (cli) or configOptions (plugins). Refer to [configuration docs](https://openapi-generator.tech/docs/configuration) for more details.
+>>>>>>> 4d371178a2f63a90aca7c35f6e5a25825f277cf3
```
Comment (wing328) [inline docs/generators/csharp-netcore-functions.md:19]: merge conflict. please have a look

File: modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/CsharpNetcoreFunctionsServerCodegen.java [hunk 2]
```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright 2018 SmartBear Software
```
Comment (wing328) [inline modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/CsharpNetcoreFunctionsServerCodegen.java:3]: I don't think this line is needed: https://opensource.stackexchange.com/questions/7300/copyright-notice-in-the-file-header-apache-v2-license

**Comment by Abrhm7786** [comment]
@wing328 Thanks for review. Made changes, checks pass now.

**Comment by Abrhm7786** [comment]
@wing328 Please could this PR be completed? Thanks. :)

**Comment by wing328** [comment]
Let me review over the weekend.

**Pull request merged**